webpack.config.js

```diff
@@ -8,6 +8,7 @@ module.exports = {
   devtool: 'eval-source-map',
   devServer: {
     contentBase: path.join(__dirname, '/src/'),
+    historyApiFallback: true,
```

Comment:
Curious, what is this for? I found the [documentation](https://webpack.github.io/docs/webpack-dev-server.html#the-historyapifallback-option) but I'm not sure what the application is.

Reply:
I was having errors when navigating the site, I would often get a sort of "cannot GET /about..." type error, and this was a solution I found when browsing around.

Reply:
Oh Webpack, you are a mysterious mistress full of mystery.